readme.adoc

```diff
@@ -94,9 +94,29 @@ Once you have installed from one of the below approaches it is recommended you r
 to have it setup your `PATH` to include jbang app scripts + it will on operating systems that supports
 it setup a `j!` alias you can use instead of `jbang`.
 
+=== Using Jbang
+
+Perhaps the coolest way to install Jbang is using Jbang itself.
```

Comment:
```suggestion
The simplest way to install `jbang` is using Jbang itself.
```

Reply:
Aww shucks, I liked "coolest" better :stuck_out_tongue:

Reply:
"Perhaps" doesn't exactly oozes confidence :)

Reply:
Well "coolness" is of course a matter of taste :-)